BackendServices/AuxiliaryServices/WebAPIService/VEEMEE/VEEMEEClass.cs

```diff
@@ -27,6 +28,13 @@ public VEEMEEClass(string method, string absolutepath)
             string result = null;
             string resultContentType = null;
 
+            if(absolutepath.Contains("//WardrobeWars/Images"))
```

Comment:
Can you add a method check? Like GET.

Reply:
I don't see why for a specific endpoint that ONLY WILL EVER PULL IMAGES... but fine.